script/Arcueid/Day 7/07_00_ARC07_3F.txt

```diff
@@ -712,7 +712,7 @@ Compared to him, the vampire lurking in this city is of a much lower purity."
 -- Page 385, Offset 17533.
 -- 「断言はできないわね。ちょっと前までのわたしなら問題なかったと思うけど、今のわたしは病み上がりだもの。敵の方が力をつけている可能性が高いわ」
 "I can't say for sure.
- If we were talking about me as of a couple days ago it'd be no problem, but right now I'm still recuperating.
+ If we were talking about me as of a couple days ago, it'd be no problem, but right now I'm still recuperating.
```

Comment:
If we were talking about me from a couple of days ago, it'd be no problem, but right now, I'm still recuperating.